pkg/universal_sigver/client.go

```diff
@@ -47,24 +47,24 @@ type Client interface {
 type backend struct {
 	provider          *ethclient.Client
 	smartWalletConfig *smart_wallet.Config
-	entryPointAddress *common.Address
+	entryPointAddress common.Address
 }
 
-func NewUniversalSigVer(providerURL string, smartWalletConfig *smart_wallet.Config, entryPointAddress *common.Address) Client {
+func NewUniversalSigVer(providerURL string, smartWalletConfig *smart_wallet.Config, entryPointAddress common.Address) (Client, error) {
 	provider, err := ethclient.Dial(providerURL)
 	if err != nil {
-		panic(fmt.Errorf("failed to connect to Ethereum node: %w", err))
+		return nil, fmt.Errorf("failed to connect to Ethereum node: %w", err)
 	}
 
 	var entryPointAddress_ = entryPointAddress
```

Comment:
```suggestion
	if entryPointAddress.Cmp(common.Address{}) == 0 {
		entryPointAddress = entryPointV0_6Address
	}
```

Reply:
We don't need entryPointAddress_ if we are not using pointers

Reply:
[20115a3](https://github.com/layer-3/clearsync/pull/268/commits/20115a32ebbaeea813cef609d9bb3eb194b82a65)